packages/tldraw/src/lib/shapes/text/TextShapeUtil.tsx

```diff
@@ -51,11 +51,12 @@ export class TextShapeUtil extends ShapeUtil<TLTextShape> {
 		return sizeCache.get(shape.props, (props) => getTextSize(this.editor, props))
 	}
 
-	getGeometry(shape: TLTextShape) {
+	getGeometry(shape: TLTextShape, context: string) {
```

Comment:
i think, looking forward, `context` might possibly end up being different, multiple kinds of things to help paint a picture. wdyt about it being an object? (with an initial property called `{originShape : 'arrow'}`)

Reply:
Makes sense. My trick for the different computed caches needs a string for the key but this could take an object.